neonvm/apis/neonvm/v1/virtualmachine_types.go

```diff
@@ -532,6 +570,8 @@ type VirtualMachineStatus struct {
 	MemoryProvider *MemoryProvider `json:"memoryProvider,omitempty"`
 	// +optional
 	SSHSecretName string `json:"sshSecretName,omitempty"`
+	// +optional
+	CurrentLogicalTime *LogicalTime `json:"currentLogicalTime,omitempty"`
```

Comment:
What does "current" mean here? (is there a better name that's more descriptive of the semantics we're promising? regardless, please add a comment explaining this)

Reply:
Just thought: maybe I should rename `LogicalTime` to `ResourcesVersion`, so it better conveys the intended semantics?

Reply:
"Current" means the associated changes have propagated through neonvm.

Reply:
Copying from a DM:

`ResourcesVersion` is very close to `ResourceVersion`, which is a k8s metadata thing. But otherwise, I think that's a good direction. Maybe something like `UpdatedTo`? or `ResourcesRequestedAt`? (trying to avoid "version" as well because it can be confused for schema version of the CRD)

Reply:
Resolved, as there isn't any logical time left.